tiup/tiup-cluster-no-sudo-mode.md

```diff
@@ -90,6 +114,10 @@ This document describes how to use the TiUP no-sudo mode to deploy a cluster.
    
 ## Manually repair failed check items
 
+> **Note:**
+>
+> If you use a minimal install, make sure the `tar` package is installed. Otherwise, the `tiup cluster check` command will fail.
```

Comment:
What does "minimal install" mean? Is there any reference link?

Reply:
I answered this in another conversation

Reply:
Do we need to update "If you use a minimal install" to "If your operation system uses a minimal install" for clarity?

Reply:
This is specific to RHEL/Rocky/Fedora/OEL/etc. Other distributions like Debian or Ubuntu don't have this (afaik).

But on all platforms with all installation methods it doesn't hurt to make sure that the `tar` package is installed.

I don't think changing this text adds clarity.

We could change it like this:
```suggestion
> If you use a minimal install during the installation of your Operating System, make sure the `tar` package is installed. Otherwise, the `tiup cluster check` command will fail.
```

But I think leaving this unchanged would be best.

Reply:
OK. I agree that leaving it unchanged is likely the best approach in this case.